Helper/Asset/AssetHelperRuntime.php

```diff
@@ -55,12 +55,26 @@ public function assets(string $hash, string $saveDir = 'bundles'): void
         }
     }
 
-    public function unzip(string $hash, string $saveDir): array
+    public function unzip(string $hash, string $saveDir, bool $mergeContent = false): array
```

Comment:
If the asset function is a emsch function (depends on the environment) the unzip function could be moved to common?

Reply:
ems_zip available in commonBundle